packages/app-builder/src/repositories/TransferRepository.ts

```diff
@@ -0,0 +1,33 @@
+import { type MarbleApi } from '@app-builder/infra/marble-api';
+import { adaptTransfer, type Transfer } from '@app-builder/models/transfer';
+
+export interface TransferRepository {
+  listTransfers(args: {
+    /**
+     * ID of the transfer in the Partner database (do not confuse with the ID generated by Marble database)
```

Comment:
TBH if the naming is too ambiguous, we could rename it - for starters in the front end repo openapi and models.

Reply:
Good idea. I'll change it in the models, but not in the `openapi` for two reasons :
1. I can't change query param naming since this will break the contract
2. I really like to be as closed as possible as backend naming (this convention proved to be really helpfull to iterate over the `openapi.yaml` along the time)